modules/os2forms_nemid/src/Plugin/WebformElement/ServiceplatformenCprElementBase.php

```diff
@@ -27,8 +28,26 @@ public function handleElementPrepopulate(array &$element, FormStateInterface &$f
       $prepopulateKey = $this->getPrepopulateFieldFieldKey($element);
       if ($value = $cprLookupResult->getFieldValue($prepopulateKey)) {
         $element['#value'] = $value;
+
+        // Appending name and address protection.
+        if (!empty($value)) {
+          $element['#value'] .= $this->appendNameAddressProtectedText($cprLookupResult);
+        }
       }
     }
   }
 
+  /**
+   * Appends name/address protected text, if person has name/address protection.
+   *
+   * @param \Drupal\os2web_datalookup\LookupResult\CprLookupResult $cprLookupResult
+   *   Initialized CprLooupResult.
+   *
+   * @return string
+   *   String " (Navne- og adressebeskyttet)" or nothing.
+   */
+  public function appendNameAddressProtectedText(CprLookupResult $cprLookupResult) {
+    return $cprLookupResult->isNameAddressProtected() ? ' (Navne- og adressebeskyttet)' : '';
```

Comment:
“(Navne- og adressebeskyttet)” should be translatable (OS2Forms support multi-lingual forms).